packages/dcos-integration-test/extra/resiliency/test_aws_cf.py

```diff
@@ -0,0 +1,75 @@
+import logging
```

Comment:
is there a way to say "this folder is resiliency tests" rather than having to individually mark any of them with badness when we forget?